### ISSUE TITLE
the `6.4.3` have an issue of `Cannot find module 'lodash/isEqualWith' `

### ISSUE THREAD

**Issue by arvinxx** [testing-library/jest-dom#598]
- `@testing-library/jest-dom` version: 6.4.3
- `node` version: 20.9.0
- `jest` (or `vitest`) version: `vitest@1.2.2`
- `npm` (or `yarn`) version: 9.2.0

### Relevant code or config:

we have tests run correctly in 6.4.2 , but failed in 6.4.3

### What you did:

npm run test

### What happened:

```
Error: Cannot find module '/Users/arvinxx/CodeProjects/LobeHub/lobe-chat/node_modules/.pnpm/@testing-library+jest-dom@6.4.3_vitest@1.2.2_@edge-runtime+vm@3.2.0_@types+node@20.12.8_happy-dom@14.7.1_terser@5.31.0_/node_modules/lodash/isEqualWith' imported from /Users/arvinxx/CodeProjects/LobeHub/lobe-chat/node_modules/.pnpm/@testing-library+jest-dom@6.4.3_vitest@1.2.2_@edge-runtime+vm@3.2.0_@types+node@20.12.8_happy-dom@14.7.1_terser@5.31.0_/node_modules/@testing-library/jest-dom/dist/index.mjs
Did you mean to import lodash@4.17.21/node_modules/lodash/isEqualWith.js?
```

you can check the test ci here: https://github.com/lobehub/lobe-chat/actions/runs/8935588080/job/24544431213

### Reproduction:

https://github.com/lobehub/lobe-chat/tree/reproduction/jest-dom-6.4.3

### Problem description:

it shows the `lodash/isEqualWith` can not be found

### Suggested solution:

I think the issue caused by the change of this: https://github.com/testing-library/jest-dom/compare/v6.4.2...v6.4.3#diff-26a76dbc98134c4a252853d189242826b97531753657672282afdd05d608dfac

it should be the `import isEqualWith from 'lodash/isEqualWith.js'`

**Comment by aurorascharff** [comment]
I encountered this as well. Inspecting my @testing-library/jest-dom/dist/node-modules, there is no lodash in here, only dom-accsesbility-api (missing also other packages) from @testting-library/jest-dom/dist/index.ts: 

require('redent');
require('@adobe/css-tools');
require('dom-accessibility-api');
require('aria-query');
require('chalk');
require('lodash/isEqualWith'); // this one is the failing one
require('css.escape');

**Comment by aurorascharff** [comment]
Looking at commits (not sure if this is relevant):

https://github.com/testing-library/jest-dom/commit/bd82f64592696c9710c59becaf1adfeb1e4c6169
![image](https://github.com/testing-library/jest-dom/assets/66901228/3fe3fd0c-1c4f-41ef-88a1-077b86328e5d)

**Comment by larsmunkholm** [comment]
Please fix 😭

**Comment by rhummelmose** [comment]
Guys come on... :D

**Comment by ddolcimascolo** [comment]
Hit by this too. Do you need any help guys?

**Comment by ddolcimascolo** [comment]
> Looking at commits: There is this 1 commit between 6.4.2 and 6.4.3 that isn't in in the release log but contains changes that might be relevant.
> 
> [bd82f64](https://github.com/testing-library/jest-dom/commit/bd82f64592696c9710c59becaf1adfeb1e4c6169)
> 
> ![image](https://private-user-images.githubusercontent.com/66901228/327697353-3fe3fd0c-1c4f-41ef-88a1-077b86328e5d.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTQ3Mjc3NzIsIm5iZiI6MTcxNDcyNzQ3MiwicGF0aCI6Ii82NjkwMTIyOC8zMjc2OTczNTMtM2ZlM2ZkMGMtMWM0Zi00MWVmLTg4YTEtMDc3Yjg2MzI4ZTVkLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA1MDMlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNTAzVDA5MTExMlomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPThlZDZmYzQwMGVkMjg1NmY4MTdhM2E5NTNkYWNkZTFiYjEzMDBmY2ViNDc0NWU5ZTMyNjRjZTBkOTc1YmYyNGImWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.O-5bN-6ieIRIwvVIb6M6c2lpr020ZplpEfh-bY_4Bng) ![image](https://private-user-images.githubusercontent.com/66901228/327701055-096d2a61-6a6c-4b8f-86a7-10a27cd3941a.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTQ3Mjc3NzIsIm5iZiI6MTcxNDcyNzQ3MiwicGF0aCI6Ii82NjkwMTIyOC8zMjc3MDEwNTUtMDk2ZDJhNjEtNmE2Yy00YjhmLTg2YTctMTBhMjdjZDM5NDFhLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA1MDMlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNTAzVDA5MTExMlomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTU0MmFjZDE2MjFhNWQyMGIzNTVmMzZiMmI0ODUwYjdmM2VmNmM0MmE3ZGE3OGUzNDQyNGExYTRlOGRkMmY3ZDImWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.ig_3TSsdp3h58KABCw2V4tymdC8HMkskTA-fFuu7GaY)

@re-taro Can you check?

**Comment by drazenbuljovcic** [comment]
Hit by this just now as well.
Obv before the issue if fixed you can pin to `6.4.2`.

**Comment by MatanBobi** [comment]
Just encountered this one too, I've created a PR. Thanks everyone for the report :)

**Comment by ddolcimascolo** [comment]
Any ETA?

**Comment by KevBeltrao** [comment]
any temporary fix?

**Comment by timothyokooboh** [comment]
Currently stuck because of this issue

**Comment by philwolstenholme** [comment]
There's no need for this to block anyone or to rush the maintainers to release a fix, just downgrade to the last working version. Google for something like 'npm install specific version'.

I am on my phone so this might not be 100% copy/pastable but try something like `npm i "@testing-library/jest-dom@6.4.2" --save-dev`

**Comment by MatanBobi** [comment]
:tada: This fix is included in version 6.4.5 :tada:

The release is available on:
- [npm package (@latest dist-tag)](https://www.npmjs.com/package/@testing-library/jest-dom/v/6.4.5)
- [GitHub release](https://github.com/testing-library/jest-dom/releases/tag/v6.4.5)

Your **[semantic-release](https://github.com/semantic-release/semantic-release)** bot :package::rocket:

**Comment by okize** [comment]
`v6.4.5` fixed the issue for me, thank you for the fast fix!
